<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FightWidget</class>
 <widget class="QWidget" name="FightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>682</x>
     <y>547</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton {
	font: 75 14pt &quot;Segoe Print&quot;;
    background-color: palegoldenrod;
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 15px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetPokemonShow" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>311</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;
background-color: rgb(255, 105, 107);


border-color: rgb(16, 16, 16);
border: 2px solid rgb(0,0,0);
border-radius : 15px;</string>
   </property>
   <widget class="QLabel" name="labelPic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;

background-color: rgb(255, 255, 255);
border-color: rgb(16, 16, 16);
border: 2px solid rgb(0,0,0);
border-radius : 1px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">border: 0px solid rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>HP</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">border: 0px solid rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelSpeed">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">border: 0px solid rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>Definse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;

background-color: rgb(255, 105, 107);
border-color: rgb(16, 16, 16);
border: 0px solid rgb(0,0,0);
border-radius : 15px;
</string>
       </property>
       <property name="text">
        <string>Attack</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelHP">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelAttack">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelDefinse">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetPokemonShow_2" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>311</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;
background-color: rgb(40, 230, 247);
border-color: rgb(16, 16, 16);
border: 2px solid rgb(0,0,0);
border-radius : 15px;</string>
   </property>
   <widget class="QLabel" name="labelPic2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;

background-color: rgb(255, 255, 255);
border-color: rgb(16, 16, 16);
border: 2px solid rgb(0,0,0);
border-radius : 1px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="1">
      <widget class="QLabel" name="labelHP2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">border: 0px solid rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>Definse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelDefinse2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">border: 0px solid rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>HP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;
background-color: rgb(40, 230, 247);
border-color: rgb(16, 16, 16);
border: 0px solid rgb(0,0,0);
border-radius : 15px;</string>
       </property>
       <property name="text">
        <string>Attack</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">border: 0px solid rgb(0,0,0);</string>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelAttack2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelSpeed2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>Your Pokemon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>Your Rival</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Segoe Print&quot;;
font: 75 24pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>VS</string>
   </property>
  </widget>
  <widget class="QListView" name="listViewRivalPokemon">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>241</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 16, 16);
border: 4px solid rgb(0,0,0);
border-radius : 15px;</string>
   </property>
  </widget>
  <widget class="QListView" name="listViewYourPokemon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>241</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;
background-color: rgb(255,255,255);
border-color: rgb(16, 16, 16);
border: 4px solid rgb(0,0,0);
border-radius : 15px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMode1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton {
	font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;
    background-color: rgb(239, 180, 121);
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 35px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
   </property>
   <property name="text">
    <string>决斗赛</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMode0">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton {
	font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;
    background-color: rgb(239, 180, 121);
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 35px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
   </property>
   <property name="text">
    <string>升级赛</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAuto">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;Microsoft YaHei UI Light&quot;;
    background-color: rgb(239, 180, 121);
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 35px;
	line=height:12pt;

text-align:center;</string>
   </property>
   <property name="text">
    <string>自动战斗</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
